connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -193,20 +189,10 @@ class SparkConnectPlanner(plan: proto.Relation, session: SparkSession) {
    */
   private def transformScalarFunction(fun: proto.Expression.UnresolvedFunction): Expression = {
     val funName = fun.getPartsList.asScala.mkString(".")
-    funName match {
-      case "gt" =>
-        assert(fun.getArgumentsCount == 2, "`gt` function must have two arguments.")
-        expressions.GreaterThan(
-          transformExpression(fun.getArguments(0)),
-          transformExpression(fun.getArguments(1)))
-      case "eq" =>
-        assert(fun.getArgumentsCount == 2, "`eq` function must have two arguments.")
-        expressions.EqualTo(
-          transformExpression(fun.getArguments(0)),
-          transformExpression(fun.getArguments(1)))
-      case _ =>
-        lookupFunction(funName, fun.getArgumentsList.asScala.map(transformExpression).toSeq)
-    }
+    UnresolvedFunction(
+      Seq(funName),
```

Comment:
Testing a negative case is great. 

I was also thinking a test here:
```
val funName = fun.getPartsList.asScala.mkString(".")

UnresolvedFunction(
      Seq(funName),
      ...
```

Basically it first concatenates the function name by `.` then passing that to a Seq which become Seq with one element, but why not just don't do the concatenation, or we have to do the  concatenation?

Having a test case here will better demonstrate the intention of this implementation.

Reply:
Will do!

Reply:
There is another thing that we probably have to do similar to what you've done for the unresolved identifier, just passing the full string to the backend and then figure out there when do we split accordingly.